chassis/sonic_platform/module.py

```diff
@@ -307,7 +309,19 @@ def set_admin_state(self, up):
         Returns:
             bool: True if the request has been successful, False if not
         """
-        return False
+        if self.get_type() == self.MODULE_TYPE_FABRIC:
+            if up == MODULE_ADMIN_DOWN:
+                logger.log_info("Power off {} module ...".format(self.module_name))
+                nokia_common._power_onoff_SFM(self.hw_slot,False)
+                logger.log_info("Chassis module {} shutdown completed".format(self.module_name))
```

Comment:
I don't see much value of this log here. Recommend removing this one.

Reply:
Fix it.